### PR TITLE
Touches up nuke messages for xenomorphs 

### PR DESCRIPTION
**Pull request by InsaneRed**
# About the pull request

Adds more xenomorph flavoured messages to nuke messages and fixes a few errors on pred part

# Explain why it's good for the game

its cooler and more hive focused rather then just "le nuke is le going off", ive kept the starting ones same just to give new players a better idea of whats happening

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

image
image

</details>

# Changelog
spellcheck: Fixes up predator ship AI nuke alerts
add: Adds new hive-flavored to the decrpytion & the initial nuke alerts
/:cl:

### CODE REVIEW

File: code/game/machinery/nuclearbomb.dm [hunk 1]
```diff
@@ -333,13 +333,13 @@ GLOBAL_VAR_INIT(bomb_set, FALSE)
 		announcement_helper("WARNING.\n\nDETONATION IN [round(timeleft/10)] SECONDS.", "HQ Intel Division", humans_other, 'sound/misc/notice1.ogg')
 		//preds part
 		var/t_left = duration2text_sec(round(rand(timeleft - timeleft / 10, timeleft + timeleft / 10)))
-		yautja_announcement(SPAN_YAUTJABOLDBIG("WARNING!\n\nYou have approximately [t_left] seconds to abandon the hunting grounds before activation of human Purification Device."))
+		yautja_announcement(SPAN_YAUTJABOLDBIG("WARNING!\n\nYou have approximately [t_left] seconds to abandon the hunting grounds before activation of the human purification device."))
 		//xenos part
 		var/warning
 		if(timer_warning & NUKE_SHOW_TIMER_HALF)
-			warning = "Hive killer is halfway through preparation cycle!"
+			warning = "A shiver goes down our carapace as we feel the approaching end...the hive killer is halfway through it's preparation cycle!"
```
Comment (Birdtalon) [inline code/game/machinery/nuclearbomb.dm:340]: its

File: code/game/machinery/nuclearbomb.dm [hunk 2]
```diff
@@ -575,27 +575,27 @@ GLOBAL_VAR_INIT(bomb_set, FALSE)
...
 		var/time_left = duration2text_sec(round(rand(decryption_time - decryption_time / 10, decryption_time + decryption_time / 10)))
-		yautja_announcement(SPAN_YAUTJABOLDBIG("WARNING!\n\nYou have approximately [time_left] seconds to abandon the hunting grounds before human Purification Device is able to be activated."))
+		yautja_announcement(SPAN_YAUTJABOLDBIG("WARNING!\n\nYou have approximately [time_left] seconds to abandon the hunting grounds before the human purification device is able to be activated."))
 
 		//xenos part
-		var/warning = "Hive killer is almost prepared to be activated!"
+		var/warning = "We are almost out of time, STOP THEM."
 		if(timer_warning & NUKE_DECRYPT_SHOW_TIMER_HALF)
-			warning = "Hive killer is halfway through its initial phase!"
+			warning = "The hive grows restless! its halfway done.."
```
Comment (Birdtalon) [inline code/game/machinery/nuclearbomb.dm:598]: it's

File: code/game/machinery/nuclearbomb.dm [hunk 3]
```diff
@@ -575,27 +575,27 @@ GLOBAL_VAR_INIT(bomb_set, FALSE)
...
-			yautja_announcement(SPAN_YAUTJABOLDBIG("WARNING!\n\nThe human Purification Device is able to be activated."))
+			yautja_announcement(SPAN_YAUTJABOLDBIG("WARNING!\n\nThe human purification device is able to be activated."))
 
 			var/datum/hive_status/hive
 			for(var/hivenumber in GLOB.hive_datum)
 				hive = GLOB.hive_datum[hivenumber]
 				if(!length(hive.totalXenos))
 					return
-				xeno_announcement(SPAN_XENOANNOUNCE("The hive killer is ready to be activated! Assault at once!"), hive.hivenumber, XENO_GENERAL_ANNOUNCE)
+				xeno_announcement(SPAN_XENOANNOUNCE("We get a sense of impending doom...the hive killer is ready to be activated."), hive.hivenumber, XENO_GENERAL_ANNOUNCE)
```
Comment (Birdtalon) [inline code/game/machinery/nuclearbomb.dm:585]: Put a space between your ... and the next word

File: code/game/machinery/nuclearbomb.dm (hunk 1 above)
Comment (Birdtalon) [reply code/game/machinery/nuclearbomb.dm:340]: space between ... and the

File: code/game/machinery/nuclearbomb.dm [hunk 4]
```diff
@@ -111,7 +111,7 @@ GLOBAL_VAR_INIT(bomb_set, FALSE)
 
 		if(isqueen(user))
 			if(timing && GLOB.bomb_set)
-				user.visible_message(SPAN_INFO("[user] begins to defuse \the [src]."), SPAN_INFO("You begin to defuse \the [src]. This will take some time..."))
+				user.visible_message(SPAN_INFO("[user] begins covering \the [src] with resin."), SPAN_INFO("You start regurgitating and overflowing the \the [src] with resin.. stopping the electronics from working, this will take some time...."))
```
Comment (Birdtalon) [inline code/game/machinery/nuclearbomb.dm:114]: ... instead of ..

**Review by Birdtalon — APPROVED** [review]

File: code/game/machinery/nuclearbomb.dm (hunk 2 above)
Comment (BroseidonMOO) [inline code/game/machinery/nuclearbomb.dm:598]: Capitalization is wrong, also why are there two periods?

File: code/game/machinery/nuclearbomb.dm (hunk 3 above)
Comment (BroseidonMOO) [reply code/game/machinery/nuclearbomb.dm:585]: Not resolved, still no space between ... and the

File: code/game/machinery/nuclearbomb.dm [hunk 5]
```diff
@@ -333,13 +333,13 @@ GLOBAL_VAR_INIT(bomb_set, FALSE)
 		announcement_helper("WARNING.\n\nDETONATION IN [round(timeleft/10)] SECONDS.", "HQ Intel Division", humans_other, 'sound/misc/notice1.ogg')
 		//preds part
 		var/t_left = duration2text_sec(round(rand(timeleft - timeleft / 10, timeleft + timeleft / 10)))
-		yautja_announcement(SPAN_YAUTJABOLDBIG("WARNING!\n\nYou have approximately [t_left] seconds to abandon the hunting grounds before activation of human Purification Device."))
+		yautja_announcement(SPAN_YAUTJABOLDBIG("WARNING!\n\nYou have approximately [t_left] seconds to abandon the hunting grounds before activation of the human purification device."))
 		//xenos part
 		var/warning
 		if(timer_warning & NUKE_SHOW_TIMER_HALF)
-			warning = "Hive killer is halfway through preparation cycle!"
+			warning = "A shiver goes down our carapace as we feel the approaching end... the hive killer is halfway through it's preparation cycle!"
```
Comment (BroseidonMOO) [inline code/game/machinery/nuclearbomb.dm:340]: its not it's

File: code/game/machinery/nuclearbomb.dm [hunk 6]
```diff
@@ -111,7 +111,7 @@ GLOBAL_VAR_INIT(bomb_set, FALSE)
 
 		if(isqueen(user))
 			if(timing && GLOB.bomb_set)
-				user.visible_message(SPAN_INFO("[user] begins to defuse \the [src]."), SPAN_INFO("You begin to defuse \the [src]. This will take some time..."))
+				user.visible_message(SPAN_INFO("[user] begins covering \the [src] with resin."), SPAN_INFO("You start regurgitating and overflowing the \the [src] with resin... stopping the electronics from working, this will take some time..."))
```
Comment (BroseidonMOO) [inline code/game/machinery/nuclearbomb.dm:114]: I don't think overflowing is the appropriate verb, this should be rephrased.

File: code/game/machinery/nuclearbomb.dm (hunk 2 above)
Comment (BroseidonMOO) [reply code/game/machinery/nuclearbomb.dm:598]: It should be it's

**Review by BroseidonMOO — CHANGES_REQUESTED** [review]

**Review by Birdtalon — APPROVED** [review]

**Pull request merged**